docs/contribute/release_checklist.md

```diff
@@ -6,20 +6,29 @@ permalink: /contributing/release-checklist
 
 ## Before the release
 
-- Make sure version number is correct or update version in Config.kt 
+- Make sure version number is correct or update version in Config.kt.
```

Comment:
Would you mind changing `Config.kt ` for ` buildconfig.yml`? :)

Reply:
Thanks, done!